superset-frontend/src/SqlLab/main.less

```diff
@@ -208,6 +208,10 @@ div.Workspace {
   .ace_content {
     height: 100%;
   }
+
+  .ace_editor * {
+    font: inherit!important;
```

Comment:
ugh, `!important` 🤮  I guess if this is the only way to fix it then it's ok

Although, do we need to apply this to all elements that are descendants of `.ace_editor` or can we be more specific here? Also, could you add a space between inherit and !important?

Reply:
It seems that the issue is around the bolded typeface in Fira Code, with the same issue with Inter and monospace. The only one I was able to get to work was "Courier New" so far. Targeting just the bold text seems to work (nested under .ace_editor.ace_editor) if you don't want to change everything:
```  
.ace_keyword {
    font-family: 'Courier New';
  }
```
we could try out a few different common system fonts if you want to try to find a sans serif.

Reply:
"Menlo" works and is sans serif, but for Mac only if you want to add that and fall back to Courier.